src/Network/Network/AzureFirewallPolicy/ExplicitProxy/NewAzureFirewallPolicyExplicitProxyCommand.cs

```diff
@@ -0,0 +1,70 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+
+using Microsoft.Azure.Commands.Network.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "FirewallPolicyExplicitProxy", SupportsShouldProcess = true), OutputType(typeof(PSAzureFirewallPolicyExplicitProxy))]
+    public class NewAzureFirewallPolicyExplicitProxyCommand : NetworkBaseCmdlet
```

Comment:
I would like recommend using New-AzFirewallPolicyExplicitProxyObject to name this command to explicitly show this command is used to create a local in-memory object.

Reply:
It was named so to be consistent with other in-memory firewall policy related objects like New-AzFirewallPolicyDnsSetting, New-AzFirewallPolicySqlSetting,  New-AzFirewallPolicyFilterRuleCollection. Please let me know if you still want me to rename it.

Reply:
Make sense. Current one is good to me as well